jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/CoreContextHandler.java

```diff
@@ -63,13 +61,14 @@ public class CoreContextHandler extends ContextHandler implements Deployable
 {
     private static final Logger LOG = LoggerFactory.getLogger(CoreContextHandler.class);
     private static final String ORIGINAL_BASE_RESOURCE = "org.eclipse.jetty.webapp.originalBaseResource";
+    private static final String[] SUPPORTED_ARCHIVE_EXTENSIONS = new String[]{"war", "jar", "zip"};
     private boolean _initialized = false;
     private List<Resource> _extraClasspath;
     private boolean _builtClassLoader = false;
 
     public CoreContextHandler()
     {
-        this("/");
+        super();
```

Comment:
Remove the constructor, there is no point just calling super().

Reply:
Done.

Reply:
Reverted this change.
This empty no-args constructor is required.

Otherwise code like ...

``` java
Object obj = clazz.getConstructor().newInstance();
```

or 

```java
Object obj = clazz.getDeclaredConstructor().newInstance();
```

will result in ...

```
java.lang.NoSuchMethodException: org.eclipse.jetty.server.handler.CoreContextHandler.<init>()
```

Reply:
Keep the constructor, but remove the call to `super()`.